index.js

```diff
@@ -32,6 +32,13 @@ If.propTypes = {
 	render: PropTypes.func
 };
 
+export const For = ({render, children, of}) => of.map((item, index) => render ? render(item, index) : children(item, index));
```

Comment:
I think we should pick on way to do it. Let's go with the `render` variant and drop the `children` variant.

Reply:
Good point, I will change it